packages/core/helper-plugin/src/utils/getFetchClient/index.js

```diff
@@ -1,12 +1,23 @@
 import instance from '../fetchClient';
+import isAbsoluteUrl from '../isAbsoluteUrl';
+import addPrependingSlash from '../addPrependingSlash';
+
+export const checkUrl = (url) => {
```

Comment:
How come we're exporting this function?

Reply:
If it's just for testing this isn't necessary, we should just add more tests to `useFetchClient` to ensure the hook does what we want, not testing the internal implementation which imo is what you're doing here with the two functions you've added and this one too :)

Reply:
I agree with Josh.

I also think the name of the function is not clear. What does `check` mean? It normalizes a URL.

Reply:
ok for both the comments, I will move the tests in the useFetchClient and change the function name